RocketChat/Rocket.Chat
# [IMPROVE] Rewrite OTR modals

## Proposed changes (including videos or screenshots)


![image](https://user-images.githubusercontent.com/40830821/124513267-cb510800-ddb0-11eb-8165-f103029c348f.png)
![image](https://user-images.githubusercontent.com/40830821/124513354-04897800-ddb1-11eb-96f4-41fe906ca0d7.png)
![image](https://user-images.githubusercontent.com/40830821/124513395-1b2fcf00-ddb1-11eb-83e4-3f8f9b4676ba.png)



## Issue(s)


## Steps to test or reproduce

- Create OTR with another user (both need to be logged in at the same time)
- New OTR acceptance modal should be displayed
- Deny OTR
- On the user's screen that created the OTR, the new "deny" modal should display
- Request OTR again
- With one of the users, end OTR
- New modal for ending OTR should show up in the other user's screen

## Further comments
